src/Queue/Broker/Redis.php

```
@@ -10,6 +10,8 @@
 
 class Redis implements Publisher, Consumer
 {
+    private const int POP_TIMEOUT = 2;
```

PHPStan (2.1.33)

Class constants with native types are supported only on PHP 8.3 and later.


(classConstant.nativeTypeNotSupported)